### ISSUE TITLE
Enabling git books option

### ISSUE THREAD

**Issue by dominijk** [volaya/gis-book#7]
Have you considered setting up the book so it can be editted on git as this? https://www.datavizforall.org/introduction/why.html

**Comment by volaya** [comment]
Yes, using GitBook would be nice. In any case, it can now be edited in git as well. Instead of markdown, I use LaTeX, and I have a script for creating the html and epub versions, so it's a similar approach. LaTeX is more complex than HTML, but the chapters do not have much LaTeX code, so it's easy to understand and edit.

Probably moving to GitBook would make things easier, but the typographical quality of the PDF output is rather poor when compared to the LaTeX one (at least in the PDFs I have seen...), and that is something I would miss.

thanks for the suggestion!

**Comment by dominijk** [comment]
Ah, i hand't realised it would be a PDF download as well as web based.
